openshift-provision/openshift-cluster-create.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: openshift-install
+spec:
+  workspaces:
+    - name: install-dir
+      mountPath: /tekton/home/
+    - name: secrets
+  params:
+    - name: PLATFORM
+      type: string
+      description: Platform name where the openshift cluster needs to be deployed such as AWS, GCP etc
+      default: ""
+    - name: PROJECT_ID
+      type: string
+      default: ""
+      description: Project Id required in case of gcp
+    - name: CLUSTER_NAME
+      type: string
+      description: Cluster name
+    - name: BASE_DOMAIN
+      type: string
+      description: Base domain of your cluster install
+    - name: REPLICAS
+      description: How many replicas for the master and worker node
+      type: string
+    - name: REGION
+      type: string
+      description: AWS Region where to install
+  steps:
+    - name: openshift-install-config
+      image: registry.access.redhat.com/ubi8/ubi-minimal:latest
+      workingDir: $(workspaces.install-dir.path)
+      script: |
+        #!/usr/bin/env bash
+
+        echo "1 $(ls -a)"
+        credsPath=""
+        projectId=""
+        if [ "$(params.PLATFORM)" == "gcp" ]; then
+          credsPath+=".gcp"
+          mkdir .gcp
+          projectId+="ProjectID: $(params.PROJECT_ID)"
+        else
+          credsPath+=".aws"
+          mkdir .aws
+        fi
+
+        cp $(workspaces.secrets.path)/* $credsPath
+
+        echo "2 $(ls -a $credsPath/)"
+
+        cat $credsPath/credentials
```

Comment:
why do you cat the credentials ? this would show them to everyone, although that may be expected but that could lead for some credential leaking

Reply:
it was a debugger. Will fix it :sweat_smile: